src/Microsoft.Health.Fhir.Core/Features/Search/Parameters/SearchParameterUtilities.cs

```diff
@@ -0,0 +1,65 @@
+﻿// -------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License (MIT). See LICENSE in the repo root for license information.
+// -------------------------------------------------------------------------------------------------
+
+using System;
+using System.Collections.Generic;
+using System.Threading.Tasks;
+using EnsureThat;
+using Hl7.Fhir.ElementModel;
+using Microsoft.Health.Fhir.Core.Exceptions;
+using Microsoft.Health.Fhir.Core.Features.Definition;
+using Microsoft.Health.Fhir.Core.Features.Definition.BundleWrappers;
+using Microsoft.Health.Fhir.Core.Features.Search.Registry;
+using Microsoft.Health.Fhir.Core.Models;
+
+namespace Microsoft.Health.Fhir.Core.Features.Search.Parameters
+{
+    public class SearchParameterUtilities : ISearchParameterUtilities
```

Comment:
Can `Utilities` be internal? Or could this be an extension on ISearchParameterDefinitionManager?

Reply:
With your other comment about hooking this functionality up using a different mediatr method, I would review the changes here along with that AB#78200.

Reply:
I agree, and Utilities sounds more like a drawer of helper methods :)